src/coverlet.console/Program.cs

```diff
@@ -185,10 +222,14 @@ static int Main(string[] args)
                 coverageTable.AddRow("Average", $"{totalLinePercent / numModules}%", $"{totalBranchPercent / numModules}%", $"{totalMethodPercent / numModules}%");
 
                 logger.LogInformation(coverageTable.ToStringAlternative());
-
+                if (process.ExitCode > 0)
```

Comment:
why not `== 1`?

Reply:
I did read dotnet tests return 0 for test succees and 1 for any failure. But to be on safer side, I went for anything > 0.. I can change it.

Reply:
For curiosity, can you try to run with "build fail" and check number?If it's different than `1` we could add and else with different type error.

Reply:
Its returning 1 if build fails. But aren't its requirement that in Console tool, we should run with `--no-build` args?

Reply:
> But aren't its requirement that in Console tool, we should run with --no-build args?

Yes I was only curious...ok keep `> 0` at the moment I don't see particular problems